README.rst

```diff
@@ -62,9 +91,30 @@ uses ``xprocess`` internally:
             timeout = 45
 
             # max lines read from stdout when matching pattern
-            # optional, defaults to 100 lines
+            # optional, defaults to 50 lines
             max_read_lines = 100
 
+            def startup_callback(self):
```

Comment:
Sorry for the bickshedding, but perhaps `startup_check` would be a better name as it conveys better the purpose of the method?

Reply:
Agreed, thanks for the tip 👍🏻